library/alloc/tests/str.rs

```diff
@@ -1978,9 +1978,14 @@ fn const_str_ptr() {
     const B: &'static [u8; 2] = &A;
     const C: *const u8 = B as *const u8;
 
-    unsafe {
+    // Miri does not deduplicate consts (https://github.com/rust-lang/miri/issues/131)
+    #[cfg(not(miri))]
```

Comment:
I'd prefer to have the comment and the attribute on the same line (so that grepping for the attribute also shows explanations) but rustfmt won't let me. :(